build_script_generator.go

```diff
@@ -121,6 +121,13 @@ func (g *webBuildScriptGenerator) Generate(ctx gocontext.Context, job Job) ([]by
 		u.User = nil
 	}
 
+	jp := job.Payload()
+	if jp != nil {
+		q := u.Query()
+		q.Add("job_id", fmt.Sprintf("%d", jp.Job.ID))
```

Comment:
Also, `strconv.Itoa(jp.Job.ID)` would avoid the `fmt` format string parsing part. I don't think it's needed for performance reason, but it might make the readability a bit better?